templates/zerver/integrations/discourse.md

```diff
@@ -1,16 +1,37 @@
-1.  {!create-an-incoming-webhook.md!}
+# Zulip Discourse integration
 
-    The API keys for "Incoming webhook" bots are limited to only
-    sending messages via webhooks. Thus, this bot type lessens
-    the security risks associated with exposing the bot's API
-    key to third-party services.
+Get Discourse notifications in Zulip!
 
-1.  Copy the `USERNAME` and `API KEY` - you'll need it later.
+{start_tabs}
 
-1.  Head over to the
-    [discourse-chat-integration setup instructions](https://meta.discourse.org/t/68501)
-    and complete them.
+1. {!create-an-incoming-webhook.md!}
 
-{!congrats.md!}
+1. Install the Discourse [Chat Integration][chat-integration].
 
-![Discourse chat integration](/static/images/integrations/discourse/001.png)
+1. In your **Discourse site settings**, set the following properties.
+
+    * Set `chat_integration_zulip_server` to your Zulip hostname, e.g.,
+      `https://your_organization.zulipchat.com`.
+    * Set `chat_integration_zulip_bot_api_key` to your Zulip bot's
+      [API key](api-key).
+    * Set `chat_integration_zulip_bot_email_address` to your Zulip bot's
+      email address.
+    * Enable `chat_integration_zulip_enabled`.
+
+1. Go to the **Plugins** tab, click on **Chat Integration**. Select
+   **Zulip**, and click **Add Channel**.
+
+1. Set **Stream** to the [channel](/help/create-a-channel) name that you'd
+   like to receive notifications in, set **Subject** to the topic name, and
+   click **Save Channel**.
+
+{end_tabs}
+
```

Comment:
Removed the example screenshot, it's just too large and dominates the entire page if left as-is.